platform/ui/index.js

```diff
@@ -10,6 +10,10 @@ export {
   Button,
   ButtonGroup,
   DateRange,
+  FilterDateRange,
+  FilterSelect,
+  FilterText,
```

Comment:
These are not necessarily limited to use with filtering. Maybe:

`InputText`
`InputSelect`
`InputDateRange`

Reply:
Liked it. I'll update.